### ISSUE TITLE
Support 32x32 framebuffers

### ISSUE THREAD

**Issue by yuriks** [citra-emu/citra#2514]
@FernandoS27 found a game that uses this: Luigi's Mansion 2.

Unknown whether this actually matters. Unless the game is doing TextureCopys or is manually modifying the framebuffer, defaulting to 8x8 mode like we do is counter-acted by DisplayTransfers also only working in 8x8 mode, which causes the final linear result to be correct.

**Comment by benderscruffy** [comment]
has this been implemented yet  ??? just wondering as its 7 years old

**Comment by ScottNash042** [comment]
@GPUCode 
has this been implemented yet ???

**Comment by GPUCode** [comment]
Nope, given that a single game uses this and it works correctly there hasn't been much interest on this tbh.
